tasks/grunt-chg.js

```diff
@@ -37,11 +37,19 @@ module.exports = function(grunt) {
 
   grunt.registerTask('chg-release', 'Add a new release and move unrleased changes under it', function(version) {
     var done =  this.async();
-    commands.release(version, {}, getCallback(done));
+    commands.release(version, {}, function(err, release) {
+      if (err) {
+        grunt.log.error(err.message);
+        done(false);
+      }
+
+      grunt.config.set('chg.release.title', release.title);
```

Comment:
I don't know exactly why but this feels dirty to me. Let's talk through the use case when you have a second.